splunk/help.md

```diff
@@ -23,7 +23,20 @@
 
 To connect to Splunk, you must have valid credentials and network access to the Splunk API port (Splunk's default is TCP/8089). This plugin supports both the Free and Enterprise Splunk licenses.
 
-If use the Splunk free license, configure your Splunk instance to allow remote login by adding the following line to the general stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
+To configure an alert trigger, the Splunk server must be able to connect back to the Komand server via http(s) ports to send webhook events.
```

Comment:
We’ll likely want to indicate that this is specific to Komand customer and if they want to do the same thing use InsightConnect app for Splunk with a link to it, that way we cover both bases.

Reply:
Looks like the trigger section was removed in help.md too in this PR

Reply:
Manually added trigger. Change the comment.